avalon/mongodb.py

```diff
@@ -378,7 +378,7 @@ def active_project(self):
 
     @requires_install
     @auto_reconnect
-    def projects(self, query_filter=None, projection=None):
+    def projects(self, query_filter=None, projection=None, no_archived=False):
```

Comment:
I would reverse the default value to `True` (in most of cases you don't want inactive projects) and change `no_archived` to `skip_archived` which more describes what will happen.

Reply:
make sense, will have that changed :)